identity/aziot-identity-common-http/src/lib.rs

```diff
@@ -50,12 +50,20 @@ pub mod get_device_identity {
 }
 
 pub mod create_module_identity {
+    #[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+    #[serde(rename_all = "camelCase")]
```

Comment:
>rename_all [](start = 12, length = 10)

You can remove rename_all if this attribute is flattened in the Request struct?

Reply:
Without the rename, the request will be:
```json
{
    "type": "local",
    "moduleId": "mqtt-server",
    "LocalIdOpts": {
        "type": "x509",
        "attributes": "server"
    }
}
```
instead of
```json
{
    "type": "local",
    "moduleId": "mqtt-server",
    "localIdOpts": {
        "type": "x509",
        "attributes": "server"
    }
}
```

So I used the rename_all to make localIdOpts consistent with the other JSON keys.